app/models/medium.rb

```diff
@@ -0,0 +1,3 @@
+class Medium < ApplicationRecord
+  validates :kind, presence: true
+end
```

Comment:
It's specified that medium has a kind/type/format attribute and a url attribute. You can also check if the url attribute is valid with a regex <3

```suggestion
class Medium < ApplicationRecord
  URL_REGEX = //

  validates :kind, presence: true
  validates :url, presence: true, format: { with: URL_REGEX, message: 'not a valid url' }
end
```

Reply:
Totally forgot the url validation 😢